client/src/components/MultiJurisdictionAudit/RoundManagement/OfflineBatchRoundDataEntry.tsx

```diff
@@ -292,7 +301,10 @@ const OfflineBatchRoundDataEntry = ({ round }: IProps) => {
                     document.getElementById('results-table')!.outerHTML
                   }
                 />
-                <Button onClick={() => setIsConfirmOpen(true)}>
+                <Button
+                  onClick={() => setIsConfirmOpen(true)}
+                  disabled={!!finalizedAt}
```

Comment:
nice catch with these. it's a shame disabling the `fieldset` doesn't disable the buttons